front/lib/swr/data_source_views.ts

```diff
@@ -54,10 +54,12 @@ export function useMultipleDataSourceViewsContentNodes({
   dataSourceViewsAndInternalIds,
   owner,
   viewType,
+  useFullParentIds,
```

Comment:
What is this used for?

Reply:
That's to skip the parents cropping. We want the selected nodes in the view, but need to fetch the parents relative to the connection, since we need to display the partial checkboxes in the tree. 
Note that we could do the call on the default system view instead 🧐

Reply:
Done - removed the useFullParentIds param and do all the logic inside VaultManagedDataSourceModal